### DISCUSSION TITLE
Cordova support in web platform

### DISCUSSION THREAD

**Discussion by sertal70** [ionic-team/capacitor#3813] [Q&A] [answered]
Hi all,
I were expecting that capacitor support for cordova plugins would cover web platform, but when I try to use a cordova plugin in the console I see the following error:

```
Native: tried calling EmailComposer.open, but Cordova is not available. Make sure to include cordova.js or run in a device/simulator
```

For what I can see, a file named `cordova.js` is present in the `node_modules/@capacitor/core` directory, so why I'm getting that error?

**Comment by jcesarmobile** [comment, marked as answer]
Capacitor doesn't support Cordova plugins on web
https://github.com/ionic-team/capacitor/issues/769
